rdmo/questions/models/question.py

```diff
@@ -1,4 +1,5 @@
 from django.conf import settings
+from django.contrib.sites.models import Site
```

Comment:
Leftover?

Reply:
no, Ive placed the `editors` field also on the `Question` ,
need to fix the `help_text` though